scripts/merge-and-build.sh

```diff
@@ -84,17 +115,37 @@ git clone git@github.com:openshift/ose.git
 cd ose
 if [ "${OSE_VERSION}" == "${OSE_MASTER}" ] ; then
 ## Remove git checkout once master is 3.6
-  git checkout -q enterprise-${OSE_VERSION}
+  git checkout -q fake-master
   git remote add upstream git@github.com:openshift/origin.git --no-tags
   git fetch --all
 
+  # TODO: Once we hard-reset master to contain Origin commits, the carry commits, and the latest tito tag commit
+  # all previous tags are going to be dropped. During the migration, I think we will need to manually tag the
+  # last tito commit or maybe not.
+  last_tag="$( git describe --abbrev=0 --tags )"
+
+  sanity_check
+  maybe_webconsole_commit="$( git log HEAD~2..HEAD~1 --pretty=%s )"
+  if [[ ${maybe_webconsole_commit} =~ "[DROP] bump origin-web-console"* ]]; then
```

Comment:
If there is no change in the web console repo for more than the timeframe between two rebases, there is no commit for the web console.